system-dependencies.sh

```diff
@@ -651,6 +667,9 @@ function check_mono () {
 		if ! test -z $PROVISION_MONO; then
 			install_mono
 			ACTUAL_MONO_VERSION=`cat $MONO_VERSION_FILE`
+		elif test -n "$OPTIONAL_MONO"; then
+			warn "It's recommended to have no higher than Mono $MAX_MONO_VERSION, found $ACTUAL_MONO_VERSION. Download URL: $MIN_MONO_URL"
```

Comment:
I just want to say for the record that I *love* how if a resource isn't available that you direct the user where to download the right version.